behaviour/graql/language/define.feature

```diff
@@ -1018,92 +928,700 @@ Feature: Graql Define Query
       | VDR |
 
 
-  Scenario: define abstract attribute type creates a type
+  Scenario: define attribute type that has itself creates an attribute
```

Comment:
better description might be "defining an attribute owning itself creates self-ownership" - it's not about the attribute, it's about the ownership that is interesting

Reply:
Yes I agree, and to be honest there are quite a number of cases where I've written "creates a relation" or "creates an attribute" as the outcome because it's what actually happens as a result of the _define_, but you're right that it's not the interesting part. I will go through and address them.